src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClientModelFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.ApplicationModel.Configuration
+{
+    public static class ConfigurationClientModelFactory
```

Comment:
Should we simply call these types ModelBuilder? i.e. each namespace would have a type with this name?

Reply:
This might be fine for App Config but I can see multiple storage clients being used in the same source file. I think we should avoid having types with the same name in multiple SKDs as a rule.